demos/link-prediction/metapath2vec-link-prediction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>Last bit of curiosity: why PCA for these? (I know it's what's used in the Node2Vec one too, so not doesn't need to change, just wondering.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1614/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Hmm yeah I don't have a good reason for it other than copying it from other demos - do you think tsne should be the "default" unless there's a good reason?